src/main/resources/templates/project.html

```diff
@@ -903,7 +906,7 @@ <h3>This repository is not active</h3>
                                 data: form.serialize(),
                                 success: function (project) {
                                     $("#loadingProject").hide();
-                                    displayProject(project.selfOwner, project);
+                                    displayProject(project.selfOwner, project, true);
```

Comment:
@criske is this the only place where we change the function call? Shouldn't we also have ``displayProject(project.selfOwner, project, false)`` somewhere? :D

Reply:
@amihaiemil In our case we ca ignore it when is false; `pmInvitePending` is evaluated as falsy when is undefined and used in the if statement `project.provider === 'github' && pmInvitePending`.

Reply:
@criske great :D